packages/angular/src/tracing.ts

```diff
@@ -90,9 +155,6 @@ export class TraceService implements OnDestroy {
       }
 
       if (activeTransaction) {
-        if (this._routingSpan) {
-          this._routingSpan.end();
-        }
         // eslint-disable-next-line deprecation/deprecation
```

Comment:
why are we removing this check? I think the idea was to end an active _routingSpan before starting a new one. 
Is this no longer a concern with the new way? And if that's the case, wouldn't we need to gate with `hooksBasedInstrumentation`?

Reply:
Oh nevermind, I missed that this was moved further up.